src/v2/Apps/Auctions/AuctionsApp.tsx

```diff
@@ -16,7 +17,28 @@ const AuctionsApp: React.FC<AuctionsAppProps> = ({ children }) => {
   return (
     <AppContainer>
       <AuctionsMeta />
-
+      <Box ml={[2, 4]}>
+        <Text pt="20px" pb="10px" variant="largeTitle">
```

Comment:
Thanks for updating these values! We use shorthand numbers that tap into our theme, thats' why one will alway's sees things like `p={1}` everywhere: https://github.com/artsy/palette/blob/master/packages/palette/src/Theme.tsx#L168-L194

This is a `key-value` mapping to our spacing-scale values:

```tsx
<Text pt={2} pb={1} variant="largeTitle">
```

Reply:
Updated! Thanks for that clarification.